<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarbEntryScreen</class>
 <widget class="QWidget" name="CarbEntryScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>909</width>
    <height>335</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="homeButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>30</y>
     <width>141</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Tandem LOGO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="backButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;-</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelCarbValue">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>16</y>
     <width>341</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>DejaVu Sans</family>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Enter Carbs</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="confirmButton">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>10</y>
     <width>83</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>âœ“</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>40</y>
     <width>62</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>grams</string>
   </property>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>511</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="btn1">
      <property name="text">
       <string>1</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btn5">
      <property name="text">
       <string>5</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QPushButton" name="btn7">
      <property name="text">
       <string>7</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="btn4">
      <property name="text">
       <string>4</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="btn2">
      <property name="text">
       <string>2</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="btnPlusMinus">
      <property name="text">
       <string>+/=</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QPushButton" name="btn8">
      <property name="text">
       <string>8</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="btn3">
      <property name="text">
       <string>3</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="btn6">
      <property name="text">
       <string>6</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btn9">
      <property name="text">
       <string>9</string>
      </property>
     </widget>
    </item>
    <item row="3" column="2">
     <widget class="QPushButton" name="btnClear">
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="btn0">
      <property name="text">
       <string>0</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
